optuna/distributions.py

```diff
@@ -319,13 +320,16 @@ class IntLogUniformDistribution(BaseDistribution):
         step:
             A step for spacing between values.
 
-            .. note::
-                This value is valid for only 1. Otherwise, the value is replaced with 1.
-
             .. warning::
                 Deprecated in v2.0.0. ``step`` argument will be removed in the future.
                 The removal of this feature is currently scheduled for v4.0.0,
                 but this schedule is subject to change.
+
+                Samplers and other components in Optuna relying on this distribution will ignore
+                this value and assume that
+                :attr:`~optuna.distributions.IntLogUniformDistribution.step` is always 1.
```

Comment:
[nit] `step` appears twice in this warning message, but the styles are not consistent. I think both of them are acceptable, but could you choose one of them?

![image](https://user-images.githubusercontent.com/3255979/85849155-66e68a80-b7e5-11ea-887d-7c8ae1e3601a.png)

Reply:
Aligned to the former. I guess it's obvious without hyperreferencing.